### PR TITLE
TOK-408: estimated rewards

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- Estimated rewards

# Link

- [TOK-408](https://rsklabs.atlassian.net/browse/TOK-408)

### CODE REVIEW

File: src/app/collective-rewards/rewards/MyRewards.tsx [hunk 1]
```diff
@@ -8,16 +8,18 @@ import {
...
 import { PricesContextProvider } from '@/shared/context/PricesContext'
-import { getCoinBaseAddress } from '@/app/collective-rewards/utils'
+import { getCoinbaseAddress } from '@/app/collective-rewards/utils'
 import { tokenContracts } from '@/lib/contracts'
 import { Popover } from '@/components/Popover'
 import { Button } from '@/components/Button'
 import { Paragraph } from '@/components/Typography'
 
 export const Rewards: FC<{ builder: Address; gauge: Address }> = ({ builder, gauge }) => {
+  console.log('🚀 ~ gauge:', gauge)
```
Comment (antomor) [inline src/app/collective-rewards/rewards/MyRewards.tsx:22]: Should remove it? or do we want to keep it for debugging purposes?

File: src/app/collective-rewards/rewards/EstimatedRewards.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    isLoading: rifLoading,
+    error: rifError,
+  } = useGetRewardMetrics(gauge, rifRewards ?? 0n, totalPotentialRewardsValue, rif.symbol, currency)
+  const {
+    data: rbtcRewardsMetrics,
+    isLoading: rbtcLoading,
+    error: rbtcError,
+  } = useGetRewardMetrics(gauge, rbtcRewards ?? 0n, totalPotentialRewardsValue, rbtc.symbol, currency)
+
+  useHandleErrors([
```
Comment (antomor) [inline src/app/collective-rewards/rewards/EstimatedRewards.tsx:78]: I'd combine these errors into one

**Review by antomor — APPROVED** [review]

File: src/app/collective-rewards/rewards/EstimatedRewards.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    [token: string]: Token
+  }
+}
+
+export const EstimatedRewards: FC<EstimatedRewardsProps> = ({ data: { rif, rbtc }, ...rest }) => {
+  return (
+    <MetricsCard borderless>
+      <MetricsCardTitle title="Estimated rewards" data-testid="EstimatedRewards" />
+      <TokenRewards {...rest} token={{ ...rif, id: 'rif' }} />
+      <TokenRewards {...rest} token={{ ...rbtc, id: 'rbtc' }} />
```
Comment (jurajpiar) [inline src/app/collective-rewards/rewards/EstimatedRewards.tsx:94]: bit hacky but I didn't want to waste time on it. feel free to improve it

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**